src/App.jsx

```diff
@@ -1,104 +1,29 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getCommentsId(commentId) {
+  return (
+    commentsFromServer.filter(comments => comments.id === commentId) || null
+  );
+}
```

Comment:
The function `getCommentsId` is filtering comments by `id`, but it seems like it should filter by `postId` to get all comments related to a specific post. Consider changing `comments.id === commentId` to `comments.postId === commentId`.